pack/fhv/fhvp.json

```diff
@@ -0,0 +1,227 @@
+    "traits": "Creature. Monster Flora. Science.",
+    "type_code": "asset",
+    "xp": 0
+  },
+  {
+    "bonded_to": "Ravenous Myconid",
+    "code": "10045",
+    "cost": 1,
+    "faction_code": "seeker",
+    "illustrator": "Edgar Sánchez Hidalgo",
+    "name": "Uncanny Growth",
+    "pack_code": "fhvp",
+    "position": 45,
+    "quantity": 1,
+    "text": "Bonded (Ravenous Myconid).\n<b>Investigate.</b> After this test resolves, place 1 resource on Ravenous Myconid, as growth,s for each point you succeed by. Set Uncanny Growth aside, out of play. If you fail, return Uncanny Growth to your hand.",
+    "traits": "Insight. Science.",
+    "type_code": "event"
+  },
+  {
+    "code": "10051",
+    "cost": 2,
+    "deck_limit": 2,
+    "faction_code": "seeker",
+    "flavor": "Every cent of that research grant counts.",
+    "illustrator": "Pixoloid Studios",
+    "name": "Well-Funded",
+    "pack_code": "fhvp",
+    "position": 51,
+    "quantity": 2,
+    "skill_wild": 1,
+    "text": "While you control a [[Science]] or [[Tool]] asset, Well-Funded gains [wild] (while you control 3 or more [[Science]] and/or [[Tool]] assets, Well-Funded gains [wild][wild], instead).",
+    "traits": "Fortune.",
+    "type_code": "skill",
+    "xp": 0
+  },
+  {
+    "code": "10072",
+    "cost": 0,
+    "deck_limit": 2,
+    "faction_code": "rogue",
+    "illustrator": "Carlos Palma Cruchaga",
+    "name": "\"I'll Pay You Back!\"",
+    "pack_code": "fhvp",
+    "position": 72,
+    "quantity": 2,
+    "skill_agility": 1,
+    "skill_willpower": 1,
+    "text": "Fast. Play only during your turn.\nChoose an investigator at your location. Move all resources from that investigator's resource pool to your resource pool. At the end of the round, move all resources from your resource pool to that investigator's resource pool.",
+    "traits": "Gambit. Trick.",
+    "type_code": "event",
+    "xp": 0
+  },
+  {
+    "code": "10110",
+    "cost": 3,
+    "deck_limit": 2,
+    "faction_code": "survivor",
+    "illustrator": "Pixoloid Studios",
+    "name": "Pitchfork",
+    "pack_code": "fhvp",
+    "position": 110,
+    "quantity": 2,
+    "skill_agility": 1,
+    "skill_combat": 1,
+    "slot": "Hand x2",
+    "text": "[action]: <b>Fight.</b> You get +1 [combat] and deal +2 damage for this attack. If this attack is successful, lose control of Pitchfork and attach it to your location. Your location gains:\n\"[action]: Take control of Pitchfork. Any investigator at Pitchfork's location may trigger this ability\"",
+    "traits": "Item. Tool. Weapon. Melee.",
+    "type_code": "asset",
+    "xp": 0
+  },
+  {
+    "code": "10111",
+    "cost": 1,
+    "deck_limit": 2,
+    "faction_code": "survivor",
+    "illustrator": "Rob Laskey",
```

Comment:
Should this be 'Robert' so that we don't end up with both Rob Laskey and Robert Laskey as illustrator entries on the db?

Reply:
Yeah, it changed on the card so I wasn't sure?  I think there have been instances of people changing their names in the past (to abbreviations or whatnot), and we've gone with it?

Reply:
Then yeah, I think we just leave it...